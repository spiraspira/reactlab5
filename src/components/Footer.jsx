import React, { Component } from "react";
import { Typography } from "@material-ui/core";

class Footer extends Component {
  render() {
    return (
      <Typography variant="body2" color="textSecondary" align="center">
        © 2024 Агентство недвижимости
      </Typography>
    );
  }
}

export default Footer;