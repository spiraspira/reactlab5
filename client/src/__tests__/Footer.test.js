import React from "react";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import Footer from "../components/Footer";

configure({ adapter: new Adapter() });

describe("Footer", () => {
  it("matches the snapshot", () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});