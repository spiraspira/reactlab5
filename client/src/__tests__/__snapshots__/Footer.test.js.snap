// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches the snapshot 1`] = `ShallowWrapper {}`;
